### ISSUE TITLE
License

### ISSUE THREAD

**Issue by Tooblippe** [Tooblippe/pandapower_gui#7]
@lthurner 

I added the same license as pandapower.
I am just not sure what to use for the copyright notice.. .currently it is for your institution.
What's your proposal here?

**Comment by lthurner** [comment]
The copyright is with whoever writes the code, so it is with you and me right now, I don't think you have to mention my institution (maybe if we get more involved later on, but right now the code is mostly yours).

About the license: PyQt is GPL licensed, so it might be mandatory to use GPL instead of BSD. From my understanding, if it only works with PyQt, we have to use a GPL license. If PyQt is only one possibliity and it is also compatible with PySide for example (which has a less strict license), we might also be able to use BSD. I am not sure however, it is a very complicated subject.

**Comment by Tooblippe** [comment]
ok... let reserve some thinking time for this.

**Comment by Tooblippe** [comment]
Maybe I should investigate PySide2?

**Comment by lthurner** [comment]
yes PySide2 is definitely a good option, because it is developed and maintained as the official Qt binding package for python. But at least last time I checked, it was not easily available yet (you have to compile the current version yourself). But since PyQt as well as PySide are only bindings for the Qt package, all names and functionality is exactly the same. So if you develop for PyQt 5, changing to PySide2 once its officially released should be very easy

**Comment by Tooblippe** [comment]
ok will investigate!
